### ISSUE TITLE
How about remove all other frontends and only leave onnx behind.

### ISSUE THREAD

**Issue by lygstate** [mil-tokyo/webdnn#885]

**Comment by milhidaka** [comment]
I think it causes some problem right now, but can be considered as future direction.
Not all operators are supported by ONNX (and export feature of each framework), so user may not be able to export their model as onnx.
WebDNN also does not implement some operators, but user can add their own implementation.

**Comment by lygstate** [comment]
@milhidaka WebDNN is a accerlation framework, and costs to much time on supporting more frontends: And we have better model converters.,

**Comment by lygstate** [comment]
@milhidaka And I found the runtime testing not not passed at all, we need focus more important things.
And webdnn doesn't supporting un-even padding size.

**Comment by milhidaka** [comment]
Thanks for suggestion.
Actually, I agree that current webdnn is relatively complex to add/configure feature, especially by new user.

**Comment by milhidaka** [comment]
WebDNN has been completely rewritten as **v2.0.0-alpha**, a browser-based ONNX inference runtime (load an ONNX model directly in the browser; no Python conversion step). The v1 Keras / Chainer / TensorFlow / Caffe converters and the old backends that this issue refers to no longer exist, so this report is no longer actionable against the current codebase.

Closing as *not planned*. Thanks for the report!
